src/app/nav/navbar.component.css

```diff
@@ -12,6 +12,19 @@
   font-size: 15px;
   box-shadow: 0 1.4px 6px 0 rgba(0,0,0,.2);
   border-top: 1px solid rgb(185, 185, 185);
+  border-radius: 0;
+}
+
+/* navbar fixed position */
+.navbar-fixed {
+  z-index: 600;
```

Comment:
z-index isn't as common as you'd think. Usually you can avoid setting z-index by changing the order of your DOM elements. When you do need to use z-index, think small. 1s and 2s are often just as good as 500 and 600.